### PR TITLE
Feature/begining code

### PR DESCRIPTION
**Pull request by Samantha-Stevens**
What does this code do?

This PR is the bulk of my code due to a strange repo within a repo issue that I encountered. 

How can this code be tested?

localhost3000/

Anything else your team should know? 

Still have clean up to do

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,111 @@
...
+
+	//Final result setter
+	const [character, setCharacter] = useState("Beth");
+
+	//const [showQuiz, setShowQuiz] = useState(false);
+	const [showCharacter, setShowCharacter] = useState(false);
+  const [banner, setBanner] = useState('')
+  const [characterInfo, setCharacterInfo] = useState([])
+
+  useEffect(() => {
```
Comment (hheyhhay) [inline src/App.js:29]: I think fetches aren't suppose to happen inside of useEffect. You can use a getData () to call the fetch and call the function in use effect.

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,111 @@
...
+    //.then(data => changeBackground(characterInfo))
+  }, []);
+  //console.log('HELLO', characterInfo)
+
+	//Sorting function
+	const answerHandler = (morty, rick, jerry, summer) => {
+    		setMorty(totalMorty + morty);
+    		setRick(totalRick + rick);
+    		setJerry(totalJerry + jerry);
+    		setSummer(totalSummer + summer);
```
Comment (hheyhhay) [inline src/App.js:42]: nice! I like how you have theses adding up and being set at the same time.

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,111 @@
...
+		}
+	};
+
+  const findImage = () => {
+		const winningCharacter = characterInfo.find((object) => {
+    return object.name.includes(character)
+  })
+	return winningCharacter.image
 }
+	const changeBackground = (name) => {
```
Comment (hheyhhay) [inline src/App.js:69]: showCharacter===true is a truthy statment so you can just do showCharacter and that will equal true

File: src/App.js [hunk 4]
```diff
@@ -1,25 +1,111 @@
...
+			setBanner(findImage());}
+		if (name === 'Jerry Smith' && showCharacter===true) {
+			setBanner(findImage());}
+		if (name === 'Summer Smith' && showCharacter===true) {
+			setBanner(findImage());}
+	};
+	//Always check which character has most points in order to reveal the respective banner
+//^^^^^This isn't working....why???
 
-export default App;
```
Comment (hheyhhay) [inline src/App.js:25]: Did you delete export Default App? You'll still need that in your index.js file.

File: src/Results.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+import './Results.css'
+
+const Results = ( { character, findImage, clearState, setShowCharacter} ) => {
+  useEffect(() => {
```
Comment (hheyhhay) [inline src/Results.js:7]: does this give you an error: React Hook useEffect has a missing dependency: ''. Either include it or remove the dependency array  react-hooks/exhaustive-deps

Mine does when I am using useEffect similarly to this. Just wondering if yours isn't you might be able to help me.

File: src/apiCalls.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+export const fetchData = async () => {
+
+  let response = await fetch('https://rickandmortyapi.com/api/character')
+
+  if (!response.ok) {
```
Comment (hheyhhay) [inline src/apiCalls.js:5]: If you want to you can have response.status === 400 and throw specific errors for 400 or 500.

File: src/index.css [hunk 7]
```diff
@@ -1,13 +1,197 @@
...
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  display: flex;
```
Comment (hheyhhay) [inline src/index.css:32]: Do you have all of your CSS in a single file? Im so curious! I can see how it would work.

**Review by hheyhhay — APPROVED** [review]

File: src/index.css (hunk 7 above)
Comment (Samantha-Stevens) [reply src/index.css:32]: Yes and surprisingly it's not the worst lol could use some cleaning up though

File: src/apiCalls.js (hunk 6 above)
Comment (Samantha-Stevens) [reply src/apiCalls.js:5]: ohhhhhh! Yes thank you! I will use this

**Pull request merged**